### ISSUE TITLE
How do I Get Facebook Access Token

### ISSUE THREAD

**Issue by djsbaa** [adrivanrex/FacebookGroupsPoster#1]
How do I Get Facebook Access Token ??

**Comment by adrivanrex** [comment]
Get your accestoken at https://developers.facebook.com/tools/explorer

**Comment by djsbaa** [comment]
I got it but the script won't work

**Comment by adrivanrex** [comment]
check the console.. press f12

**Comment by adrivanrex** [comment]
http://screencloud.net/v/xKa65 check your access token settings.. enable pages..

**Comment by djsbaa** [comment]
yeah thas what I did but the problem is the login and the publish button still disabled

**Comment by adrivanrex** [comment]
http://screencloud.net/v/cztL4 check the console...

**Comment by djsbaa** [comment]
No not working but Thank you don't break you head with me !! 👍

**Comment by adrivanrex** [comment]
Just updated the app.. it shows if the token is ok 👍
